### DISCUSSION TITLE
Ask ai for autosuggestions

### DISCUSSION THREAD

**Discussion by Haiz14** [fish-shell/fish-shell#9809] [Q&A] [answered]
In a nutshell
- store the user command text in variable 
- make api call to localhost:3000 with the user_command
-  show the return of api call as autosuggestions

It will be more like asking an LLM for the suggestions.

Stuck on this part, how to edit the autosuggestions provided by fish?

**Comment by faho** [comment, marked as answer]
Autosuggestions aren't configurable and we're not planning on adding anything. Sorry.

**Comment by Haiz14** [comment]
On a side note: Edit the fish_hustory and autosuggestions will be update in the next session.

**Reply by Haiz14** [reply]
Then use history_merge
